ColorzCore/EAInterpreter.cs

```diff
@@ -130,14 +130,33 @@ public void Interpret()
                     }
                     line.WriteData(myROM);
                 }
+
                 myROM.WriteROM();
+                return true;
             }
             else
             {
                 serr.WriteLine("Errors occurred; no changes written.");
+                return false;
             }
         }
 
+        public bool WriteNocashSymbols(TextWriter output)
+        {
+            if (myParser == null)
+            {
+                return false;
+            }
+
+            foreach (var label in myParser.GlobalScope.Head.LocalLabels())
+            {
+                // TODO: more elegant offset to address mapping
```

Comment:
TODO: -hardwareOffset flag.

Reply:
(You don't have to address this in this PR but I'm just noting that I do want to do that down the line; it'd help Zane out a lot for one)

Reply:
May I note that if you want to help Zane/support SNES memory maps you'd need to have something fancier than a simple `-hardwareOffset` flag (you'd need to be able to map different sections of the output binary to different sections of the memory map, not just one).

(I also don't know how much Zane *needs* help since his assembler (64tass) basically looks like turbo EA to me).